### PR TITLE
fuzz: enable imports in differential fuzzing 

### PR DESCRIPTION
**Pull request by mathetake**

### CODE REVIEW

**Comment by mathetake** [comment]
found the failing case with the locally run fuzz

```
(module
  (import "" "s" (memory (;0;) 0 5))
  (import "" "" (global (;0;) funcref))
  (global (;1;) (mut f64) f64.const -0x1.1ff6861000008p-652 (;=-0.00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006019210420568196;))
  (global (;2;) (mut funcref) global.get 0)
  (global (;3;) (mut i32) i32.const 1000)
  (export "\00\00\00\00\00\00\14\14\14\14\14" (memory 0))
  (export "" (global 0))
  (export "2" (global 1))
  (export "3" (global 2))
  (elem (;0;) declare externref (ref.null extern) (ref.null extern) (ref.null extern))
  (data (;0;) "\ff\ff\ff\ff")
)
```

**Comment by mathetake** [comment]
rebased onto #888

**Comment by mathetake** [comment]
ok now looks stable with the fix in #888

File: internal/integration_test/fuzz/wazerolib/nodiff.go [hunk 1]
```diff
@@ -77,6 +114,107 @@ func requireNoDiff(wasmBin []byte, requireNoError func(err error)) {
 	}
 }
 
+func ensureDummyImports(r wazero.Runtime, origin *wasm.Module, requireNoError func(err error)) (skip bool) {
```
Comment (codefromthecrypt) [inline internal/integration_test/fuzz/wazerolib/nodiff.go:117]: what "dummy imports" means might be useful to document here.

**Review by codefromthecrypt — APPROVED** [review]
love to see death of TODO

**Pull request merged**